api/client/commands.go

```diff
@@ -2722,6 +2732,7 @@ func (s *containerStats) Display(w io.Writer) error {
 
 func (cli *DockerCli) CmdStats(args ...string) error {
 	cmd := cli.Subcmd("stats", "CONTAINER [CONTAINER...]", "Display a live stream of one or more containers' resource usage statistics", true)
+	noStream := cmd.Bool([]string{"-no-stream"}, false, "Disable streaming stats and only pull the first result")
```

Comment:
That's right, but we usually do it at release time rather than for each CLI change it seems (ping @jfrazelle as 1.6.0 release captain: is it ok for you?)